contracts/message-based/PriceAwareOwnable.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./PriceAware.sol";
+
+contract PriceAwareOwnable is PriceAware, Ownable {
+
+  address private trustedSigner;
+
+  function getTrustedSigner() public view virtual returns (address) {
```

Comment:
I'm not sure if we need this method if we handle the logic in isSignerAuthorised. 
It doesn't add much value.

Reply:
Sure